glide.lock

```diff
@@ -30,15 +30,14 @@ imports:
   - private/protocol/rest
   - private/protocol/restxml
   - private/protocol/xml/xmlutil
-  - private/waiter
   - service/s3
   - service/s3/s3iface
   - service/s3/s3manager
   - service/sts
 - name: github.com/benbjohnson/clock
   version: 7dc76406b6d3c05b5f71a86293cbcf3c4ea03b19
 - name: github.com/bsm/sarama-cluster
-  version: 82cfa5798a3898e4cf7d913f4a448911af83c3f5  
+  version: 2ae74ad075d4f088751b8c074f7e88c1020d7687
```

Comment:
@GuillaumeBailey should we pin sarama to a release or is it ok to go with master for now and update this?